lib/easypost/client.rb

```diff
@@ -20,35 +20,39 @@ def initialize(api_key:, read_timeout: 60, open_timeout: 30, api_base: 'https://
 
     # Make an HTTP client once, reuse it for all requests made by this client
     # Configuration is immutable, so this is safe
-    @http_client = EasyPost::HttpClient.new(base_url, http_config)
+    @http_client = EasyPost::HttpClient.new(api_base, http_config)
   end
 
   SERVICE_CLASSES = [
     EasyPost::Services::Address,
     EasyPost::Services::ApiKey,
     EasyPost::Services::Batch,
+    EasyPost::Services::BetaReferralCustomer,
     EasyPost::Services::Billing,
     EasyPost::Services::CarrierAccount,
     EasyPost::Services::CustomsInfo,
     EasyPost::Services::CustomsItem,
+    EasyPost::Services::EndShipper,
+    EasyPost::Services::Event,
+    EasyPost::Services::Insurance,
+    EasyPost::Services::Order,
+    EasyPost::Services::Parcel,
+    EasyPost::Services::Pickup,
+    EasyPost::Services::Rate,
+    EasyPost::Services::ReferralCustomer,
   ].freeze
 
-  # Loop over the SERVICE_CLASSES to automatically define the instance variable instead of manually define it
+  # Loop over the SERVICE_CLASSES to automatically define the method and instance variable instead of manually define it
   SERVICE_CLASSES.each do |cls|
     define_method(EasyPost::InternalUtilities.to_snake_case(cls.name.split('::').last)) do
       instance_variable_set("@#{EasyPost::InternalUtilities.to_snake_case(cls.name.split('::').last)}", cls.new(self))
     end
   end
 
   # Make an HTTP request
-  #
-  # @param method [Symbol] the HTTP Verb (get, method, put, post, etc.)
-  # @param endpoint [String] URI path of the resource
-  # @param body [Object] (nil) object to be dumped to JSON
-  # @raise [EasyPost::Error] if the response has a non-2xx status code
```

Comment:
issue: I don't feel removing these are necessary - they were valid comments

Reply:
If we decide to keep these comments, we can have a separate ticket to overhaul the docstring for Ruby CL. Thought?

Reply:
I mean sure, come back to the rest in a separate PR. But they were already here so putting them back isn't something that needs to happen separately?

Reply:
added them back and also updated the docstring to match the parameters